src/implementations/twig/components/button/button.story.js

```diff
@@ -11,117 +9,162 @@ import dataCall from '@ecl/specs-component-button/demo/data--call';
 import dataGhost from '@ecl/specs-component-button/demo/data--ghost';
 import dataSearch from '@ecl/specs-component-button/demo/data--search';
 
+import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 import button from './button.html.twig';
 import notes from './README.md';
 
 const iconsList = {};
-iconsList.none = '';
-
 uiIcons.forEach((icon) => {
```

Comment:
A bit prematurely extracted and could be put inline as it's used only once. 
In case of considering a one-liner, check this example

```
const arr = ['a','b','c'];
const res = arr.reduce((a,b)=> (a[b]=b,a),{});
```

Reply:
I don't think we need it, control can be reset to default state. this is why I removed the none option.